lib/register_sources_bods/register/statement_loader.rb

```diff
@@ -28,11 +26,11 @@ def load_statements(statement_ids, load_children: true)
 
       attr_reader :statement_repository, :statements_mapper
 
-      def load_statements_children(statements)
+      def load_statements_children(statements, max_levels:)
```

Comment:
Perhaps this should default to `MAX_LEVELS` to give same behaviour as before. But if all cases using it have been covered, then it's okay.